app/controllers/api/service_dialogs_controller.rb

```diff
@@ -18,7 +18,8 @@ def fetch_service_dialogs_content(resource)
     end
 
     def create_resource(_type, _id, data)
-      DialogImportService.new.import(data)
+      dialog = DialogImportService.new.import(data)
+      fetch_service_dialogs_content(dialog)
```

Comment:
I'm still a bit confused here, the fetch_<type>_<attr> are getters for virtual attributes. Why isn't the hook on line 3 getting this for us.  set_additional_attributes asks for content.